lib/ipmi_sensor.c

```diff
@@ -169,7 +169,25 @@ ipmi_sensor_print_fc_discrete(struct ipmi_intf *intf,
 	}
 
 	if (csv_output) {
-		/* NOT IMPLEMENTED */
+        printf("%s", sr->s_id);
```

Comment:
Please format according to project's [rules](https://github.com/ipmitool/ipmitool/wiki/Coding-Standards).